.github/workflows/pip.yml

```diff
@@ -2,9 +2,9 @@ name: "Build wheels and deploy"
 
 on:
   push:
-  release:
-    types:
-      - published
+  # release:
+  #   types:
+  #     - published
```

Comment:
This event is only triggered by a published release, no need to comment for debugging (the `push:` event is triggered anyhow, so you're changing nothing)